pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java

```diff
@@ -282,4 +284,17 @@ private CompletableFuture<Void> validateClustersAsync(TenantInfo info) {
             }
         });
     }
+
+    private CompletableFuture<Void> validateAdminRoleAsync(TenantInfoImpl info) {
+        // empty adminRole shouldn't be allowed
+        if (!info.getAdminRoles().isEmpty()) {
+            for (String adminRole : info.getAdminRoles()) {
+                if (StringUtils.isBlank(adminRole) || StringUtils.containsWhitespace(adminRole)) {
```

Comment:
It indeed is an inappropriate usage to contain white space. But what error will be caused?  
I fear this could be a break change because some users may use a role with white space in their system.

If you can explain the error this could cause, it will be easy to accept this change.

Reply:
I think white space should be allowed, as long as it's not at the beginning or the end. Because if there are two role 'role1' and '         role1             ', it's confusing